syntaxes/TeX.tmLanguage.json

```diff
@@ -110,7 +110,7 @@
 					"name": "punctuation.definition.function.tex"
 				}
 			},
-			"match": "(\\\\)(?:[A-Za-z@]+|[,;])",
+			"match": "(\\\\)(?:[\\p{Alphabetic}@]+|[,;])",
```

Comment:
It seems that the Plain TeX allows any non-ASCII characters `\P{ASCII}` here while LaTeX at least respects Unicode whitespaces. For example,

Plain TeX is fine with
```tex
\def\UPlusSixteenEighty {U+1680}
\UPlusSixteenEighty 
```
but LaTeX will throw missing `\begin{document}`. Since the TeX grammar is inherited by LaTeX, here we use `\p{Alphabetic}` so that the highlight works for both TeX and LaTeX.